modules/apps/29-fee/keeper/keeper.go

```diff
@@ -144,14 +144,29 @@ func (k Keeper) GetFeeInEscrow(ctx sdk.Context, packetId *channeltypes.PacketId)
 	return fee, true
 }
 
+// IterateChannelFeesInEscrow iterates over all the fees on the given channel currently escrowed and calls the provided callback
+// if the callback returns true, then iteration is stopped.
+func (k Keeper) IterateChannelFeesInEscrow(ctx sdk.Context, portID, channelID string, cb func(identifiedFee types.IdentifiedPacketFee) (stop bool)) {
```

Comment:
nit: maybe a way to refactor GetFeeInEscrow or this function so they aren't as redundant